packages/runtime/runtime-utils/src/remoteFluidObjectHandle.ts

```diff
@@ -51,7 +51,7 @@ export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
 				url: this.absolutePath,
 				headers: {
 					[RuntimeHeaders.viaHandle]: true,
-					[RuntimeHeaders.metadata]: this.metadata,
+					[RuntimeHeaders.placeholder]: this.placeholder,
```

Comment:
this is a good example of why keeping it in an object is more extendible, as it keeps this code general rather than specific per property